### PR TITLE
Fix misspellings on an industrial scale (#1393)Co-authored-by: Tomas Votruba <tomas.vot@gmail.com>

### PR DESCRIPTION
**Pull request by szepeviktor**
Viktor stops the party of Tomas hugging typos :)

I leave it 100% up to you.
This is an inspirational PR!!

### CODE REVIEW

File: resources/posts/2018/2018-04-23-how-to-slowly-turn-your-symfony-project-to-legacy-with-action-injection.md [hunk 1]
```diff
@@ -347,7 +347,7 @@ There are 2 ways ho to avoid this completely and still use your framework:
 -
 Paul M. Jones has written [many posts about Action-Domain-Responder](https://paul-m-jones.com/adr)
 - another approach is [RequestHandler](https://jenssegers.com/goodbye-controllers-hello-request-handlers)
-- my favorite approach that [Symfony](https://symfony.com/doc/current/controller/service.html#invokable-controllers) and [Laravel](https://dyrynda.com.au/blog/single-action-controllers-in-laravel) support by default for a long time are **invokable controllers**, also called *single action controllers*
+- my favorite approach that [Symfony](https://symfony.com/doc/current/controller/service.html#invokable-controllers) and [Laravel](https://dyrynda.com.au/blog/single-action-controllers-in-laravel) support by default for a long time are **invocable controllers**, also called *single action controllers*
```
Comment (TomasVotruba) [inline resources/posts/2018/2018-04-23-how-to-slowly-turn-your-symfony-project-to-legacy-with-action-injection.md:350]: Should be `invokable` by convention

File: resources/posts/2019/2019-06-24-do-you-use-php-codesniffer-and-php-cs-fixer-phpstorm-plugin-you-are-slow-and-expensive.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ The last benefit is wired to our brains.
 
 <img src="/assets/images/posts/2019/plugin/porn.jpg" class="img-thumbnail">
 
-Slack notification, Instagram likes, underline of *typoes* and suggestions in your IDE.<br>
+Slack notification, Instagram likes, underline of *typos* and suggestions in your IDE.<br>
```
Comment (TomasVotruba) [inline resources/posts/2019/2019-06-24-do-you-use-php-codesniffer-and-php-cs-fixer-phpstorm-plugin-you-are-slow-and-expensive.md:32]: On purpose :laughing:

File: resources/posts/2020/2020-12-21-5-new-combos-opened-by-symfony-52-and-php-80.md [hunk 3]
```diff
@@ -275,7 +275,7 @@ return function (ContainerConfigurator $containerConfigurator): void {
 };
 ```
 
-❌ string `"some_param"` can create typoes, use constant instead
+❌ string `"some_param"` can create typos, use constant instead
```
Comment (TomasVotruba) [inline resources/posts/2020/2020-12-21-5-new-combos-opened-by-symfony-52-and-php-80.md:278]: On purpose :+1:

File: resources/posts/2020/2020-12-21-5-new-combos-opened-by-symfony-52-and-php-80.md [hunk 4]
```diff
@@ -132,9 +132,9 @@ Note: Are you using action injection? Migrate to promoted properties and [stop n
 
 <br>
 
-What is an invokable controller? It's a form of Symfony controller that is impossible to turn into a huge legacy controller with 50 actions.
+What is an invocable controller? It's a form of Symfony controller that is impossible to turn into a huge legacy controller with 50 actions.
```
Comment (TomasVotruba) [inline resources/posts/2020/2020-12-21-5-new-combos-opened-by-symfony-52-and-php-80.md:135]: Should be K

File: resources/posts/2020/2020-08-17-how-to-get-rid-of-magic-static-and-chaos-from-latte-filters.md [hunk 5]
```diff
@@ -413,7 +413,7 @@ My question is: can we do better...?
 
 **Update 1 month later with new option:**
 
-## 6. Invokable Filter Providers
+## 6. Invocable Filter Providers
```
Comment (TomasVotruba) [inline resources/posts/2020/2020-08-17-how-to-get-rid-of-magic-static-and-chaos-from-latte-filters.md:416]: Should be K

**Comment by TomasVotruba** [comment]
Neat :+1: Thanks :)

**Comment by szepeviktor** [comment]
> Neat 👍 Thanks :)

I leave it 100% up to you.
It is _your_ blog.

**Comment by szepeviktor** [comment]
It is an inspiration for you. To use a spelling tool like https://github.com/crate-ci/typos

**Comment by szepeviktor** [comment]
All right... I'll be your employee for 1 commit!

**Comment by szepeviktor** [comment]
Searched for all _Invocable_-s ...

**Comment by szepeviktor** [comment]
Done 🍏 

Close or Merge!

File: resources/posts/2019/2019-02-21-how-we-migrated-from-nette-to-symfony-in-3-weeks-part-1.md [hunk 6]
```diff
@@ -57,7 +57,7 @@ In the end, [Rector](https://getrector.org) helped us with many following change
 
 - Response and Request ✅
 - Presenter to Controller ✅
-- RouterFactory to Controller Annotation ✅ - REST Routes and Invocable Controllers included
+- RouterFactory to Controller Annotation ✅ - REST Routes and Invokable Controllers included
```
Comment (TomasVotruba) [inline resources/posts/2019/2019-02-21-how-we-migrated-from-nette-to-symfony-in-3-weeks-part-1.md:60]:
```suggestion
- RouterFactory to Controller Annotation ✅ - REST Routes and Invocable Controllers included
```

**Comment by TomasVotruba** [comment]
You got it :wink:

**Comment by szepeviktor** [comment]
Sometime you like _Invocable_. My head goes 🤯

**Pull request merged**